File: app/src/androidTest/java/com/battlelancer/seriesguide/modules/TestTraktModule.java
package com.battlelancer.seriesguide.modules;

import com.uwetrottmann.trakt5.TraktV2;
import com.uwetrottmann.trakt5.services.Episodes;
import com.uwetrottmann.trakt5.services.Movies;
import com.uwetrottmann.trakt5.services.Search;
import com.uwetrottmann.trakt5.services.Shows;
import com.uwetrottmann.trakt5.services.Sync;
import com.uwetrottmann.trakt5.services.Users;

public class TestTraktModule extends TraktModule {

    @Override
    Episodes provideEpisodes(TraktV2 trakt) {
        return null;
    }

    @Override
    Movies provideMovies(TraktV2 trakt) {
        return null;
    }

    @Override
    Shows provideShows(TraktV2 trakt) {
        return null;
    }

    @Override
    Search provideSearch(TraktV2 trakt) {
        return null;
    }

    @Override
    Sync provideSync(TraktV2 trakt) {
        return null;
    }

    @Override
    Users provideUsers(TraktV2 trakt) {
        return null;
    }
}

File: app/src/androidTest/java/com/battlelancer/seriesguide/modules/TestTraktModule.kt
// SPDX-License-Identifier: Apache-2.0
// Copyright 2017-2024 Uwe Trottmann

package com.battlelancer.seriesguide.modules

import com.battlelancer.seriesguide.traktapi.SgTrakt
import com.uwetrottmann.trakt5.services.Episodes
import com.uwetrottmann.trakt5.services.Movies
import com.uwetrottmann.trakt5.services.Search
import com.uwetrottmann.trakt5.services.Shows
import com.uwetrottmann.trakt5.services.Sync
import com.uwetrottmann.trakt5.services.Users

class TestTraktModule : TraktModule() {
    override fun provideEpisodes(trakt: SgTrakt): Episodes? {
        return null
    }

    override fun provideMovies(trakt: SgTrakt): Movies? {
        return null
    }

    override fun provideShows(trakt: SgTrakt): Shows? {
        return null
    }

    override fun provideSearch(trakt: SgTrakt): Search? {
        return null
    }

    override fun provideSync(trakt: SgTrakt): Sync? {
        return null
    }

    override fun provideUsers(trakt: SgTrakt): Users? {
        return null
    }
}